code/assets/js/sw.template.js

```diff
@@ -0,0 +1,50 @@
+self.oninstall = event => {
+	event.waitUntil(
+		caches.open({{NAME}}).then(cache => {
+			return cache.addAll([{{filesToCache}}]);
```

Comment:
Let's do the same as with `{{NAME}}`.

```suggestion
			return cache.addAll([{{FILES}}]);
```